zerver/views/user_groups.py

```diff
@@ -93,7 +91,7 @@ def add_user_group(
 
 
 @require_member_or_admin
-@has_request_variables
+@typed_endpoint_without_parameters
 def get_user_group(request: HttpRequest, user_profile: UserProfile) -> HttpResponse:
```

Comment:
This can be a follow-up if I end up merging this PR, but `get_user_group` seems like the wrong name for this function? `get_all_user_groups` seems much more appropriate.